tests/impl.cpp

```diff
@@ -3117,7 +3117,22 @@ result_t test_mm_blendv_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int32_t *_a = (const int32_t *) impl.mTestIntPointer1;
+    const int32_t *_b = (const int32_t *) impl.mTestIntPointer2;
+#if defined(__clang__)
+    int count = (uint64_t) _b[0] % 32;
+#else
+    int count = (uint32_t) _b[0] % 32;  // Other value greater than 31
```

Comment:
The value should not be `32` since the result would be 0~31.
The comment should be `Other value greater than 32`.

Reply:
it should be greater equal than 32 or greater than 31

Reply:
Nope.
Greater than `31` means `32` is acceptable, which is not.

I think we have different definition of the comment.
For your definition, it should be `// For generating other value greater than 31`, which means the count variable.
For my definition, it should be `// The value for doing the modulo should be greater than 32`, which means the constant for doing the modulo.

Pick a comment and give a more detailed description.